core-bundle/src/Routing/ResponseContext/CoreResponseContextFactory.php

```diff
@@ -114,6 +121,35 @@ public function createContaoWebpageResponseContext(PageModel $pageModel): Respon
             )
         ;
 
+        if ($pageModel->enableCsp) {
+            $directives = $this->cspParser->parseHeader(trim((string) $pageModel->csp));
+            $directives->setLevel1Fallback((bool) $pageModel->cspLevel1Fallback);
```

Comment:
This I would remove, like mentioned before.

Reply:
Removed in 13715f08117a19c8fdcf0e6d06034cbd61212edb